modules/apps/commerce/commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/search/spi/model/index/contributor/CommercePaymentEntryModelDocumentContributor.java

```diff
@@ -42,7 +42,7 @@ public void contribute(
 				"currencyCode", commercePaymentEntry.getCurrencyCode());
 			document.addKeyword(
 				"externalReferenceCode",
```

Comment:
as this is commerce related doesn't have access to the CPField.EXTERNAL_REFERENCE_CODE ?